### ISSUE TITLE
Verify local blobs

### ISSUE THREAD

**Issue by lkatalin** [sigstore/sigstore-rs#83]
From @lukehinds 

Currently sigstore-rs verifies a cosign signature stashed in an OCI registy, but we might also want to attest blobs locally (exist in rekor and signed with an ODIC account or pub key that we provide). We would likely want a client to perform this, but we expose the ABI in sigstore-rs to allow them to so so

**Comment by mlieberman85** [comment]
I have a PR for this for "cosign" but be aware that there's currently a bug in Rekor: https://github.com/sigstore/rekor/issues/582 so you can't actually verify the signature from Rekor. You will need to keep the signature locally.

**Comment by lkatalin** [comment]
> I have a PR for this for "cosign" but be aware that there's currently a bug in Rekor: [sigstore/rekor#582](https://github.com/sigstore/rekor/issues/582) so you can't actually verify the signature from Rekor. You will need to keep the signature locally.

Thanks for pointing that out! Looks like the issue is now resolved.

**Comment by mlieberman85** [comment]
Yep, just got merged. Not 100% sure when it'll be released though.
